### PR TITLE
Add stage that makes 2pt plots without theory

### PR DESCRIPTION
**Pull request by joezuntz**
This is work by @jgpaine that I'm uploading.

I'll suggest some changes below.

### CODE REVIEW

File: txpipe/twopoint_plots.py [hunk 1]
```diff
@@ -84,31 +83,7 @@ def run(self):
 
         for fig in outputs.values():
             fig.close()
-
```
Comment (eacharles) [inline txpipe/twopoint_plots.py:87]: Just curious, what were these lines doing?

**Review by eacharles — APPROVED** [review]
Looks good.

**Comment by joezuntz** [comment]
Ack, sorry, requested your review on completely the wrong page! This one isn’t quite done yet and needs some changes. The ceci one is what I meant to do!

**Comment by joezuntz** [comment]
Replaced with #280

**Pull request closed without merging**